tests/test_plugin.py

```diff
@@ -31,3 +38,18 @@ def test_stop_plugin_configuration_on_conn_error(mocked_get):
     expect(mocked_config._reportportal_configured is False,
            'The value of the _reportportal_configured is not False.')
     assert_expectations()
+
+
+@patch('pytest_reportportal.RPLogger.handle')
+def test_logger_handle_attachment(mock_handler, logger):
+
```

Comment:
Would it be possible to add docstrings for new functions/methods? I know it wasn't done in the past, but the sooner we start doing this the better.

Reply:
Fixed